schema/src/main/jade-structs/read_structure.struct.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "read_structure",
```

Comment:
I found that read structure is a structured object in ENCODE's raw file schema. This is a mirror of that structure. It might be overkill to parse->serialize back to string; we could instead stringify the raw JSON objects and achieve the same results with less chance of things breaking. Thoughts?

Reply:
It feels like the question is if Encode changes things, do we want it to throw an error during our transformation? Or do we want to just accept the new data format? If this is likely to be used in search, maybe we should opt for the consistent format.

Do you know if this something people would be likely to search by?